content/physical_properties/physical_properties_dielectric_permittivity.rst

```diff
@@ -57,7 +57,7 @@ Parameters used to define the dielectric properties of materials and their assoc
 +-------------------------+-----------------------------------+------------------+
 | Dielectric Permittivity | :math:`\varepsilon`               | F/m              |
 +-------------------------+-----------------------------------+------------------+
-| Electric Susceptibility | :math:`\chi_e`                    | *Unitsless*      |
+| Electric Susceptibility | :math:`\chi_e`                    | *Unitless*      |
```

Comment:
@yangdikun: the issue is here... subtle, but there needs to be an extra space: the vertical line needs to line up with the plus sign, so one space needs to be inserted